deploy/modules/website/dns_record.tf

```diff
@@ -1,7 +1,13 @@
 resource "aws_route53_record" "website_record" {
   zone_id = data.aws_route53_zone.zone.id
   name    = var.env == "prod" ? var.route53_zone_name : "${var.env}.${var.route53_zone_name}"
-  type    = "CNAME"
-  ttl     = "300"
-  records = [aws_cloudfront_distribution.website_distribution.domain_name]
+  type    = "A"
+  //ttl     = "300"
```

Comment:
What is the default TTL ?

Reply:
It is the default TTL for the resource, according to AWS docs and we cannot change it either way https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-choosing-alias-non-alias.html